coursemapper-kg/recommendation/app/services/course_materials/recommendation/resource_recommender.py

```
@@ -88,257 +125,139 @@
+        resources_dict = rrh.rank_resources(resources=resources, weights=factor_weights, 
+                                            top_n_resources=top_n_resources, recommendation_type=recommendation_type,
+                                            pagination_params=rec_params["pagination_params"]
+                                        )
+        result_final = {    
+                            "recommendation_type": rec_params["recommendation_type"],
+                            "concepts": rec_params["concepts"], 
+                            "nodes": resources_dict
+                        }
+        return result_final
+
+    def _get_resources(self, data_rec_params: dict, data_default: dict=None, top_n = 5):
```

Ruff (0.12.2)

Missing return type annotation for private function `_get_resources`

(ANN202)